app/controllers/miq_policy_controller.rb

```diff
@@ -617,16 +621,14 @@ def replace_right_cell(nodetype, replace_trees = [], presenter = nil)
       presenter.update(:main_div, r[:partial => 'condition_details', :locals => {:read_only => true}])
       right_cell_text = if @condition.id.blank?
                           _("Adding a new %{model}") % {:model => ui_lookup(:model => 'Condition')}
-                        else
-                          if @right_cell_text == @edit
-                            _("Editing %{model} Condition \"%{name}\"") %
+                        elsif @edit
```

Comment:
Sorry for the noise.  The bugfix PR got merged, that commit no longer shows here.